libs/blocks/merch/merch.js

```diff
@@ -600,6 +600,7 @@ export async function getModalAction(offers, options, el) {
   const hash = setCtaHash(el, checkoutLinkConfig, offerType);
   let url = checkoutLinkConfig[columnName];
   if (!url && !el?.isOpen3in1Modal) return undefined;
```

Comment:
shouldn't we just stop here to !url ? here if 
(!url && el?.isOpen3in1Modal) we pass this line, we pass 603 and we break 604